src/twisted/newsfragments/9553.bugfix

```diff
@@ -0,0 +1 @@
+twisted.internet.iocpreactor.FileHandle.loseConnection now calls stopWriting.
```

Comment:
Can you phrase this in more behavior-like terms rather than just "it calls this method"?  How would it misbehave before, how does it behave now?